lib/hmmlearn/tests/test_variational_categorical.py

```diff
@@ -99,6 +111,7 @@ def test_n_features(self, implementation):
             sequences = sequences.astype(float)
             assert_log_likelihood_increasing(model, sequences, lengths, 10)
 
+
```

Comment:
pep8: one blank line between methods, not two

Reply:
thanks.